Sources/Rownd/Models/Automations/AutomationTypes.swift

```diff
@@ -17,12 +17,68 @@ public struct RowndAutomation: Hashable {
     public var rules: [RowndAutomationRuleUnknown]
     public var triggers: [RowndAutomationTrigger]
     public var platform: RowndAutomationPlatform
+    
+    static let defaultEncodedInstance = RowndAutomation(
+        id: "default_id",
+        name: "faield_to_encode",
+        template: "no_template",
+        state: .disabled,
+        actions: [],
+        rules: [],
+        triggers: [],
+        platform: .web
+    )
 }
 
 extension RowndAutomation: Codable {
     enum CodingKeys: String, CodingKey {
         case id, name, template, state, actions, rules, triggers, platform
     }
+    
+    public init(from decoder: Decoder) throws {
+        do {
+            let container = try decoder.container(keyedBy: CodingKeys.self)
+            id = try container.decode(String.self, forKey: .id)
+            name = try container.decode(String.self, forKey: .name)
+            template = try container.decode(String.self, forKey: .template)
+            state = try container.decode(RowndAutomationState.self, forKey: .state)
+            actions = try container.decode([RowndAutomationAction].self, forKey: .actions)
+            rules = try container.decode([RowndAutomationRuleUnknown].self, forKey: .rules)
+            triggers = try container.decode([RowndAutomationTrigger].self, forKey: .triggers)
+            platform = try container.decode(RowndAutomationPlatform.self, forKey: .platform)
+        } catch {
+            // If decoding fails, use the default instance
+            var defaultInstance = RowndAutomation.defaultEncodedInstance
```

Comment:
Is there a way to discard / ignore the unparsable automation vs. using a fallback default?

Reply:
Updated to discard unparsable automation schemas